lib/pages/campaign_form_page.dart

```diff
@@ -101,18 +104,24 @@ class _CampaignFormPageState extends State<CampaignFormPage> {
                 const SizedBox(
                   height: 20,
                 ),
+                const SizedBox(
+                  height: 20,
+                ),
+                const CustomImagePicker(),
```

Comment:
Add `onImageIsSelected` method which return the image file:
```
CustomImagePicker(
  onImageIsSelected: (imageFile) => {
    setState(() => _imageFile = imageFile)
  }
)
```

Reply:
check it out